cmd/tk/export.go

```diff
@@ -36,6 +36,9 @@ func exportCmd() *cli.Command {
 		"lower": func(s string) string {
 			return strings.ToLower(s)
 		},
+		"mkdir": func() string {
```

Comment:
I'd like to explore what happens when we name  this function `/`,  so that you write `{{/}}`.

As `text/template` does not have arithmetic (`/` unused), this should work, right?

Reply:
`/``as function name is invalid:
`panic: function name "/" is not a valid identifier`